source/concepts/transaction.rst

```diff
@@ -153,24 +153,15 @@ If valid, the harvester stores the transaction in a block and reaches the **conf
 Continuing the previous example, the transaction gets processed and the amount stated gets transferred from the signer's account to the recipient's account.
 Additionally, the :doc:`transaction fee <fees>` is deducted from the signer's account.
 
-The transaction has **zero confirmations** at this point.
+The transaction has zero confirmations at this point.
 When another block is added to the blockchain, the transaction has one confirmation.
 The next block added to the chain will give it two confirmations and so on.
 
-.. _rollbacks:
+Under certain circumstances, like network failure or partition, the most recent confirmed blocks need to be reversed.
+:ref:`Rollbacks <rollbacks>` are characteristic in blockchain systems and essential to resolve the forks of the blockchain.
```

Comment:
:ref:`Rollbacks <rollbacks>` are characteristic of blockchain systems and essential to resolve the forks of the blockchain.